### ISSUE TITLE
unable to connect via ssh

### ISSUE THREAD

**Issue by javixeneize** [dmayer/idb#67]
Hi

I am unable to connect to my jailbroken device via ssh

Connecting via usb, it is trying to do the connection in a high port (5700)?, so the connection is rejected (open port is 22)

Connecting through wifi, it tries to connect to port 22, but the connection is rejected (i have setup my own ssh password, so the generic alpine password doesnt work)

I can connect to the device via ssh through terminal...

Any idea?

If it helps, in the device tab my device is not listed

Thanks

**Comment by rustymagnet3000** [comment]
Did you install SSH via Cydia?  This has to be installed before you can do anything with `idb`.

**Comment by javixeneize** [comment]
Wow... this is still alive!

**Comment by dmayer** [comment]
yes it is :) Sorry I was also not responsive at all. Going through all the old issues. Do you still have this problem?

**Comment by javixeneize** [comment]
I dont use the tool anymore but i think it would be great to adapt it to
the current threats. Do you feel like its a good idea?

**Comment by dmayer** [comment]
definitely. Do you have any suggestions?

**Comment by javixeneize** [comment]
Sure... a lot. Surface attack has changed a lot and also methodology. we
can spend some time on this... how can i contact you?

**Comment by dmayer** [comment]
Hm, DM me on Twitter: https://twitter.com/DanlAMayer  and can go from there?
